packages/core/src/pill/Pill.css

```diff
@@ -49,6 +49,7 @@
 
 /* Style applied to Pill on focus */
 .saltPill:focus-visible {
+  cursor: var(--salt-cursor-active);
```

Comment:
I don't think this is correct, focus styles should match hover

Reply:
Also focus-visible is keyboard specific, not sure why we'd be changing cursors here